e2e-tests/cypress/support/appeal-header-footer/confirmGoogleAnalyticsLinkIsPresent.js

```diff
@@ -2,7 +2,7 @@ module.exports = () => {
   cy.get('head script[data-cy="Google Analytics"]').should(
     'have.attr',
     'src',
-    'https://www.googletagmanager.com/gtag/js?id=G-RTYZW789M0',
+    'https://www.googletagmanager.com/gtag/js?id=googleID',
```

Comment:
Wrap in `if/else`

Reply:
To clarify in my mind then:

1. test env shoudl check that the entire GA tag doesn't exist
2. we need a *smoke test* to confirm that the GA tag does exist only in prod

That's not a statement, that's a Q around my understanding.

Reply:
the e2e-tests is in @wip at the moment because we need to see this in production asap. 
As soon as we are able to configure e2e-tests with env variable we'll check only in production that the tag is present with the right id (and maybe absence in dev)

Reply:
> To clarify in my mind then:
> 
> 1. test env shoudl check that the entire GA tag doesn't exist
> 2. we need a _smoke test_ to confirm that the GA tag does exist only in prod
> 
> That's not a statement, that's a Q around my understanding.

we're basically saying 
* the e2e test didn't really add a great deal of value
* switch it off and flush this through to production as quickly as we can so we can see things working
* //TODO put the test back in once we've solved the problem of running tests that understand the env-vars of the environment they're testing

in short we're trying to be pragmatic on this one; its a bit naughty but- yassine has *promised* to put am improved test in when we have the tech available later in the sprint ;)